08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep

```diff
@@ -0,0 +1,37 @@
+@description('Specify the name of automation account to be created.\r\nUse only alphanumerics and hyphens.\r\nThe name must not start or end with alphanumeric and must be 6 - 50 characters in length.\r\n')
+param automationAccountName string
```

Comment:
This bicep script is able to create automation account with only one variable right. Did we think of supporting multiple variables with account creation or just add variables for the existing account?

Reply:
Yeah, so the expectation from the template is to provide the valid property value. And even one type of variable is sufficient to convey this.